playbooks/deploy_edpm_os_configure.yml

```diff
@@ -64,9 +64,21 @@
         tasks_from: configure.yml
       tags:
         - edpm_timezone
+    - name: configure edpm_frr
```

Comment:
Task name should be start with an uppercase letter (see https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/rules/name.py#L127)